def handle_error():
    return "I'm sorry, I couldn't process your request. Please try again."
